lychee.toml

```diff
@@ -2,6 +2,7 @@ exclude = [
     # these are private repositories, presumably any link to it are in internal documentation only.
     "^https://github.com/near/near-ops",
     "^https://github.com/near/nearcore-private",
+    "^https://github.com/PagodaPlatform/",
```

Comment:
Not all repos in `PagodaPlatform` are private, maybe it would be better to exclude only `https://github.com/PagodaPlatform/tf-near-node` which is currently failing?